mtp_cashbook/templates/cashbook/new_credits.html

```diff
@@ -222,7 +223,7 @@ <h2 class="heading-large">{% trans 'Have you updated NOMIS?' %}</h2>
                 {% endblocktrans %}
                 <span class="mtp-none-selected">{% trans 'You haven’t selected any to process yet.' %}</span>
                 <span class="mtp-some-selected">
-                  <strong><span class="js-SelectionCount"></span></strong>{% trans 'credits selected for processing in NOMIS' %}.
+                  <strong><span class="js-SelectionCount"></span></strong>{% trans ' credits selected for processing in NOMIS' %}.
```

Comment:
the space ought to be outside the `trans` tag or else it won't be translated correctly